Documentation/CodeSnippets/FileStructure/ReadmeRstStandalone.rst.txt

```diff
@@ -16,8 +16,9 @@
     ## Usage
     ..
 
-    |                  | URL                                                           |
-    |------------------|---------------------------------------------------------------|
-    | **Repository:**  | https://<vcs-repository>                                      |
-    | **Read online:** | https://docs.typo3.org/p/<package-name>/main/en-us/           |
-    | **TER:**         | https://extensions.typo3.org/extension/<extension-key>/       |
+    .. csv-table::
+        :header: "", "URL"
+
+        **Repository:**,        https://<vcs-repository>
```

Comment:
Have you rendered this locally? Is the URL rendered as a link or not? I suspect, that this may be linked. If yes, enclose the URLs in the `:samp:` role, for example:

    :samp:`https://<vcs-repository>`

Reply:
https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/CheatSheet.html?highlight=samp#preventing-links

That is why it is in the :samp: text role

Reply:
I have rendered it for testing purpose. See develop branch of macina_searchbox 

I have added not the REST code "samp' around the links.

Reply:
This does not work with ':samp:':

See
https://github.com/franzholz/macina_searchbox/tree/develop